src/main/java/anto/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package anto;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class represents a Deadline task.
+ */
+public class Deadline extends Task {
+
+    /**
+     * DateTime that the Task must be completed by.
+     */
+    protected LocalDateTime by;
+
+    /**
+     * Creates a Deadline task with specified description and by date.
+     * @param description Description of the deadline task.
```

Comment:
Should there be an empty line between between description and parameter section?